apps/src/templates/studioHomepages/ActionBlocksWrapper.jsx

```diff
@@ -0,0 +1,93 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {LinkButton} from '@cdo/apps/componentLibrary/button';
+import {
+  Heading2,
+  OverlineTwoText,
+  BodyThreeText,
+} from '@cdo/apps/componentLibrary/typography';
+import styles from './actionBlocksWrapper.module.scss';
+
+const OneColumnActionBlock = ({
+  overline,
+  imageUrl,
+  heading,
+  description,
+  buttons,
+}) => {
+  return (
+    <div className={styles.oneColumnActionBlock}>
+      <div className={styles.contentWrapper}>
+        {overline && (
+          <OverlineTwoText className={styles.overline}>
+            {overline}
+          </OverlineTwoText>
+        )}
+        {heading && (
+          <Heading2 visualAppearance="heading-md">{heading}</Heading2>
+        )}
+        {imageUrl && <img src={imageUrl} alt="" />}
+        {description && <BodyThreeText>{description}</BodyThreeText>}
+      </div>
+      <div className={styles.buttonWrapper}>
+        {buttons &&
+          buttons.map((button, index) => (
+            <LinkButton
```

Comment:
😍 for using the new `LinkButton` component!

Reply:
I was so pumped about this too! (Thank you cc @levadadenys! ❤️ )